app/src/main/java/com/babylon/wallet/android/domain/model/assets/AssetPrice.kt

```diff
@@ -42,16 +40,14 @@ data class FiatPrice(
                 val javaCurrency = Currency.getInstance(currency.name)
                 NumberFormat.getCurrencyInstance().apply {
                     currency = javaCurrency
-                    if (price < 1.0 && price != 0.0) {
-                        maximumFractionDigits = MAX_FRACTION_DIGITS
+                    if (price == 0.0) {
+                        maximumFractionDigits = NO_PRECISION
```

Comment:
this was something that I forgot to do it last time 👀

Reply:
when actual value is 0, e.g. an account with no resources we show "0", instead of "0.00"